modules/programs/gitui/gitui.nix

```
@@ -0,0 +1,74 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+
+  cfg = config.programs.gitui;
+
+  yamlFormat = pkgs.formats.yaml { };
+
+  inherit (pkgs.stdenv.hostPlatform) isDarwin;
+
+in {
+  meta.maintainers = [ maintainers.mifom ];
```

```suggestion
  meta.maintainers = [ hm.maintainers.mifom ];
```

Home Manager maintains it's own list of maintainers under `lib.hm.maintainers`.